fah_visualizer.py

```diff
@@ -79,3 +79,11 @@ def feat(irow):
 traj_fn = output_trajectory
 backup(traj_fn)
 traj.save(traj_fn)
```

Comment:
What filetype does `traj.save` output as? I think it would be good to clarify and see if you can't save it back as an xtc + topology file for compressed, convenient viewing